build_scripts/rpm/azure-cli.spec

```diff
@@ -9,8 +9,7 @@
 %define name           azure-cli
 %define release        1%{?dist}
 %define version        %{getenv:CLI_VERSION}
-%define source_sha256  %{getenv:CLI_DOWNLOAD_SHA256}
-%define source_url     https://azurecliprod.blob.core.windows.net/releases/azure-cli_packaged_%{version}.tar.gz
+%define repo_path      %{getenv:REPO_PATH}
```

Comment:
Too bad that relative paths are not supported, which otherwise would save an environment variable

Reply:
I added it so user doesn't have to run script from root of repo.
I should be able to remove it though. Will take a look in my next PR on improvements to release process.